### ISSUE TITLE
list_filter not working in Python

### ISSUE THREAD

**Issue by azhadsyed** [duckdb/duckdb#4244]
### What happens?

I can't use `list_filter` with the Python API :(

### To Reproduce

Run this in a Python environment:

```python3
import duckdb

con = duckdb.connect()
results = con.execute("""
SELECT [1] AS l, list_filter([1], l -> l > 1)
""")
```

You will receive this error:

```
RuntimeError: Catalog Error: Scalar Function with name list_filter does not exist!
Did you mean "list_first"?
LINE 2: SELECT [1] AS l, list_filter([1], l -> l > 1)
```

### OS:

Big Sur 11.4

### DuckDB Version:

0.4.0

### DuckDB Client:

Python

### Full Name:

Azhad Syed

### Affiliation:

Personal

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

**Comment by taniabogatsch** [comment]
Hi, `list_filter` was added very recently and is not yet included in our official releases (in this case 0.4.0.). You can use the latest master branch, it should work there. And of course it will be part of the next official release! :)

**Comment by azhadsyed** [comment]
Ahh thanks! Forgot that the docs default to the current version
